src/QMCWaveFunctions/BsplineFactory/SplineR2R.h

```diff
@@ -71,16 +72,33 @@ class SplineR2R : public BsplineSet
   hContainer_type myH;
   ghContainer_type mygH;
 
+  IndexType m_true_norbs;
```

Comment:
I don't think we need this.

Reply:
Removed this and used OrbitalSetSize in base class instead.